docs/dashboards/multiple-series.md

```diff
@@ -41,7 +41,7 @@ Note: you won’t be able to add another saved question to multi-series visualiz
 
 ## Combining two saved questions
 
-If you already have two or more saved questions you’d like to compare, and they share a dimension, they can be combined onto a single dashboard card. You can even compare questions that pull data from different databases. Here’s how:
+If you already have two or more saved questions you’d like to compare, and they have the same first dimension, they can be combined onto a single dashboard card. You can even compare questions that pull data from different databases. Here’s how:
```

Comment:
not a new 51 thing but might as well clarify it now while I'm reviewing this doc...